--- a/jni/tkimg/compat/libtiff/libtiff/tif_config.h
+++ b/jni/tkimg/compat/libtiff/libtiff/tif_config.h
@@ -188,7 +188,7 @@
 #define PACKBITS_SUPPORT 1
 
 /* Support Pixar log-format algorithm (requires Zlib) */
-#define PIXARLOG_SUPPORT 1
+/* PIXARLOG_SUPPORT disabled (iwish arm64) */
 
 /* Define to necessary symbol if this constant uses a non-standard name on
    your system. */
@@ -265,7 +265,7 @@
 /* #undef X_DISPLAY_MISSING */
 
 /* Support Deflate compression */
-#define ZIP_SUPPORT 1
+/* ZIP_SUPPORT disabled (iwish arm64) */
 
 /* Enable large inode numbers on Mac OS X 10.5.  */
 #ifndef _DARWIN_USE_64_BIT_INODE
